tasks/main.yml

```diff
@@ -132,7 +132,11 @@
   become: "no"
 
 - name: "Generate special sampledata transfers"
-  command: "~/archivematica-sampledata/createtransfers.py create-variously-encoded-files"
+  command: "~/archivematica-sampledata/createtransfers/createtransfers.py {{ item }}"
+  with_items:
+    - "create-variously-encoded-files"
+    - "create-deep-transfers"
+    - "create-large-test-transfers"
```

Comment:
@mamedin `create-large-test-transfers` may generate too much data, I don't think it should be in the defaults!